proto/cosmos/group/v1/tx.proto

```diff
@@ -306,11 +302,16 @@ message MsgSubmitProposal {
   // whether it should be executed immediately on creation or not.
   // If so, proposers signatures are considered as Yes votes.
   Exec exec = 5;
+
+  // title is the title of the proposal.
```

Comment:
Can we add Since comments here and on all new fields? Also, let's do the same on the gov proto fields

Reply:
This was done everywhere but expect in this file :D